### DISCUSSION TITLE
⟪关于用户信息界面和专栏的若干建议⟫

### DISCUSSION THREAD

**Discussion by CaseYwY** [Richasy/Bili.Uwp#1220] [想法]
<img width="879" alt="屏幕截图 2022-06-23 010429" src="https://user-images.githubusercontent.com/93791824/175096703-d1891014-7eed-4bc2-977e-59003f97e5fb.png">

①希望能在用户信息界面和个人动态里增加专栏的显示，有些up发了专栏在哔哩里却没法看见

<img width="316" alt="屏幕截图 2022-06-23 010633" src="https://user-images.githubusercontent.com/93791824/175096718-09a9cfd4-6b60-46e3-a5e9-2ff7b54ec4b3.png">

②希望能在用户信息界面添加  标签，最多播放和最多收藏等功能
<img width="703" alt="屏幕截图 2022-06-23 011350" src="https://user-images.githubusercontent.com/93791824/175097800-66a5953f-0dbd-4bda-97e6-84b3d04ced45.png">

③希望能在关闭页面旁，添加一个按钮可以让专栏像视频一样满界面显示，并在设置中添加专栏的默认显示方式的选项。

现在的显示方式多少有些憋屈，大屏幕看起来实在费力

<img width="1192" alt="屏幕截图 2022-06-23 011814" src="https://user-images.githubusercontent.com/93791824/175098550-c8a271e8-4459-4b39-91d5-986bcb15c45d.png">

④希望评论区能发表情

（注：已看完关于开发方向的调整，还是希望能将其完善到位，让这作品更加完美）

**Comment by blibilijojo** [comment]
额，，你也不发到议题里，你发这里有个p用啊，我帮你发一下把
